inst/include/population_dynamics/fleet/fleet.hpp

```diff
@@ -165,8 +190,35 @@ namespace fims_popdy {
                 }
                 for (size_t a = 0; a < this->nages; a++) {
                     size_t i_age_year = y * this->nages + a;
-                    this->proportion_catch_numbers_at_age[i_age_year] = this->catch_numbers_at_age[i_age_year] / sum;
+                    this->proportion_catch_numbers_at_age[i_age_year] = 
+                    this->catch_numbers_at_age[i_age_year] / sum;
+                }
+            }
+        }
 
+        /**
+         * Evaluate the proportion of catch numbers at length.
```

Comment:
I think this-> looks great! the syntax and index is a little hard for my tiny brain to track but i think it all checks out. 

a couple clarifying questions/thoughts:
1) for a fishery this would happen after fishing and fishery selectivity has been imposed right? similarly for a  survey, this is happening after Q and survey selectivity? just want to make sure that's what "catch_numbers_at_age" is
2) in terms of priority, fleet specific age_length_conversion_matrix capabilities might be important. just a consideration. i know that is different than the SS3 approach where growth is at the population level...

Reply:
1. Yes, the catch numbers at age are the fleet specific removals in a given year which this is converting to ages. 

2. These conversion matricies are fleet specific so they could be different for each fleet in which case they would be implying implicit length selectivity differences in addition to and age selectivity difference between fleets.

Reply:
1. Yes this is directly using the model estimates for removed landings (fleets) or observed (survey) numbers at age and then distributing to length bins.
2. This is set up for fleet-specific age to length conversion so these could be different for each fleet/survey. Using different values would facilitate incorporating empirical length selectivity on top of the age selectivity.